lib/static/new-ui/features/visual-checks/components/VisualChecksPage/index.tsx

```diff
@@ -78,6 +79,46 @@ export function VisualChecksPage(): ReactNode {
         setImageChanged(true);
     }, [treeData]);
 
+    const visibleNamedImageIds = treeData.allTreeNodeIds;
+    const currentNamedImageIndex = visibleNamedImageIds.indexOf(currentTreeNodeId);
+
+    const onPrevNextImageHandler = useCallback((step: number): void => {
+        const nextIndex = currentNamedImageIndex + step;
+        if (nextIndex < 0 || nextIndex >= visibleNamedImageIds.length) {
+            return;
+        }
```

Comment:
why don't we allow the user to switch from the first screenshot/retry to the last one?